lib/commands/performance.js

```diff
@@ -188,6 +189,11 @@ commands.mobileStopPerfRecord = async function (opts = {}) {
                       `You can use 'instruments -s' command to see the list of all available profiles.`);
   }
 
+  if (!_.isEmpty(savePath)) {
+    log.info(`Found perf trace record '${localPath}'. Move to path '${savePath}'.`);
+    return await fs.rename(localPath, savePath);
```

Comment:
this will corrupt the returned result. I'd rather keep zipped trace file to save space

Reply:
Ok, the return type should be a string . 
If users want to save the trace file, they will zip the file to save space by themselves.

Reply:
the return type is base64-encoded string, not just a string